Makefile

```diff
@@ -0,0 +1,13 @@
+# Makefile
+
+# Default variables
+DOCKERFILE_PATH ?= Dockerfile
+SCYLLA_BENCH_VERSION ?= tags/v0.1.22
```

Comment:
Not sure it is good idea to have a default for the scylla-bench version.
It is easy to get unexpected s-b version in a test.

Reply:
is there a way to get the latest easily?

Reply:
There is no `latest` tag.
I made this statement assuming this workflow is going to be used with specific PRs.
So, if you are sure that specific PRs won't get problems having this default then ok, keep it as is.

Reply:
It is going to be used with specific PRs, I just thought that in most cases we want to use the latest tag and it would be easier to update it in one place here than in every place that is going to use this Makefile

Reply:
"master" branch should solve that need, if I understood everything correctly.